src/librustc_typeck/diagnostics.rs

```diff
@@ -380,6 +380,23 @@ fn main() {
 ```
 "##,
 
+E0045: r##"
+Variadic parameters are only allowed in extern "C" code. Example of
```

Comment:
This could be expanded on. "Rust only supports variadic parameters for interoperability with C code in its FFI. As such, variadic parameters can only be used with functions which are using the C ABI."